code/modules/mob/living/silicon/robot/component.dm

```diff
@@ -179,10 +179,12 @@
 		camera.status = powered
 
 /datum/robot_component/camera/install()
+	installed = 1

```

Comment:
For future ref.
`..()` instead of this, for both install and uninstall, would have been better. That way, they cleanly execute the install proc's original definition, before doing whatever the child needs to do in specific.

Better that way, just in case the install procedure changes in the future.